build/azure-pipelines.yml

```diff
@@ -65,6 +65,14 @@ jobs:
         publishRunAttachments: true
       continueOnError: true
       condition: always()
+    - task: RichCodeNavIndexer@0
```

Comment:
I assume we don't want to run this for both debug *and* release.  probably should have a condition so that it only runs on debug?